raaghu-components/src/rds-comp-assistance/rds-comp-assistance.tsx

```
@@ -149,6 +149,7 @@ const isFormValid = isNameValid(assistance?.name) && isContactNumberValid(assist
                 handleDataChanges(e.target.value, "message");
               }}
               reset={inputReset}
```

ESLint

[error] Expected indentation of 32 spaces but found 14.

(indent)